modules/SFML/audio.cppm

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file audio.cppm
+ * @brief Module file for SFML audio operations.
+ */
+
+export module sfml.audio;
+
+export import sfml.audio.InputSoundFile;
+export import sfml.audio.Listener;
+export import sfml.audio.Music;
+export import sfml.audio.OutputSoundFile;
+export import sfml.audio.PlaybackDevice;
+export import sfml.audio.Sound;
+export import sfml.audio.SoundBuffer;
+export import sfml.audio.SoundBufferRecorder;
+export import sfml.audio.SoundChannel;
+export import sfml.audio.SoundFileFactory;
+export import sfml.audio.SoundFileReader;
+export import sfml.audio.SoundFileWriter;
+export import sfml.audio.SoundRecorder;
+export import sfml.audio.SoundSource;
+export import sfml.audio.SoundStream;
```

Comment:
Please add trailing newlines to all files. I see you omitted that in a lot of these new files.